cra-tf-validate-ignore-rules.json

```diff
@@ -0,0 +1,16 @@
+{
+    "scc_rules": [
+        {
+            "scc_rule_id": "rule-8cbd597c-7471-42bd-9c88-36b2696456e9",
+            "description": "Check whether Cloud Object Storage network access is restricted to a specific IP range",
+            "ignore_reason": "Tracking in https://github.ibm.com/GoldenEye/issues/issues/3571",
```

Comment:
for consistency with other modules, update this to:

This module supports restricting network access using Context Based Restrictions (CBRs), however SCC does not yet support scanning for CBR rules, hence the rule currently fails. SCC CBR support is being tracked in AHA SCC-961